src/buffer.rs

```diff
@@ -582,7 +582,6 @@ impl PyBuffer {
             let ptr = &*self.0 as *const ffi::Py_buffer as *mut ffi::Py_buffer;
             ffi::PyBuffer_Release(ptr)
         };
-        mem::forget(self);
```

Comment:
So what on earth this `forget` mean?
Since the pointer that `self` has is already released, there's no reason for keeping it.

Reply:
It means that `drop` shouldn't be called